examples/vault-cluster-private/outputs.tf

```diff
@@ -56,4 +56,28 @@ output "ssh_key_name" {
 
 output "vault_cluster_size" {
   value = "${var.vault_cluster_size}"
-}
\ No newline at end of file
+}
+
+output "launch_config_name_servers" {
+  value = "${module.consul_cluster.launch_config_name}"
+}
+
+output "iam_role_arn_servers" {
+  value = "${module.consul_cluster.iam_role_arn}"
+}
+
+output "iam_role_id_servers" {
+  value = "${module.consul_cluster.iam_role_id}"
+}
+
+output "security_group_id_servers" {
+  value = "${module.consul_cluster.security_group_id}"
+}
+
+output "consul_cluster_cluster_tag_key" {
+  value = "${module.consul_cluster.cluster_tag_key}"
+}
+
+output "consul_cluster_cluster_tag_value" {
+  value = "${module.consul_cluster.cluster_tag_value}"
+}
```

Comment:
What are all these new outputs?

Reply:
see: https://github.com/hashicorp/terraform-aws-consul/blob/master/outputs.tf

these came from the consul project. brings the outputs to parity with that one.